.travis.yml

```diff
@@ -18,10 +18,10 @@ matrix:
     services:
       - docker
     script:
-      - ./gradlew -s -i check && ./gradlew_win -s -i check
+      - ./gradlew -s -i check build
   - os: osx
     script:
-      - ./gradlew -s -i check
+      - ./gradlew -s -i clean build
```

Comment:
No `check`?

Reply:
`build` will assemble and run tests which is nice to run as a separate step on ci from other checks, but there are none so I didn't add any.  Can reverse this though 👍